### DISCUSSION TITLE
Automatically exposing stories problem

### DISCUSSION THREAD

**Discussion by ghost** [storybookjs/storybook#15434] [Help]
I have chart story where I want to dynamically get args and file names from other folder and then I create story for each of them. Here is my code:

![image](https://user-images.githubusercontent.com/34492112/123933693-e677de00-d992-11eb-9a82-51f1d8fd675d.png)

When I run storybook localy everything works fine, but while trying to build with chromatic I am getting this message: 

![image](https://user-images.githubusercontent.com/34492112/123934010-2ccd3d00-d993-11eb-81bc-c475f4eea1bc.png)

The problem is related to export part.
Is it beacuse of destructuring assigment? How could I export stories to make it work on production?

**Comment by shilman** [comment]
As far as I know this is not ESM; the stories have to be statically named.

what do you think about this syntax:

```js
import chartsData from './Charts';

export const FuelChartDISTANCE = { args: chartsData.FuelChartDISTANCE };
export const FuelChartTIME = { args: chartsData.FuelChartTIME };
...
```
